docs/helm/kratos.md

```diff
@@ -55,48 +56,69 @@ $ helm install -f ./path/to/kratos-config.yaml ory/kratos
 
 Additionally, the following extra settings are available:
```

Comment:
```suggestion
## Helm chart configuration
Additionally, the following extra settings are available:
```

Reply:
I think we could benefit from adding an extra header in the docs, and point to the full list of helm values and their description (so the readme of the chart)

Reply:
Done :), let me know what you think.